packages/ra-data-simple-prisma/src/permissions/canAccess.ts

```diff
@@ -33,11 +33,18 @@ export const canAccess = ({
 }: {
   action: Action;
   permissions: Permissions<string>;
-  resource: string; //better type is Permission["resource"] but ra's resource is string
+  resource: string;
   record?: any;
   field?: string;
 }): boolean => {
-  if (!permissions || permissions.length === 0) return false;
+  if (!permissions || permissions.length === 0 || !resource) return false;
+
+  // Support resource.field pattern
+  if (!field && resource.includes(".")) {
+    const parts = resource.split(".");
+    resource = parts[0];
+    field = parts.slice(1).join(".");
```

Comment:
why this? a field cannot have dots in most dbs, so `a.b.c` will have a field b.c - do you mean being able to drill into a relationship?

Reply:
yes, so user.password will be converted into resource: user, and field: password.

Reply:
please explain me why `.join(".")` on the remaining parts - `password` (the remaining parts(s)) has no dots so what's the point of joining with dots again

Reply:
Ok I'll just the take parts[1] and then

Reply:
`const [resource, field] = resource.split(".");` syntax would be awesome

Reply:
good idea